packages/dappmanager/src/params.ts

```diff
@@ -151,16 +151,21 @@ const params = {
   ETH_MAINNET_RPC_URL_REMOTE:
     process.env.ETH_MAINNET_RPC_URL_REMOTE || "https://web3.dappnode.net",
 
-  // Validators legacy versions
+  // Validators legacy versions: Prysm, Prysm-prater, Prysm-gnosis
   minimumAllowedPackageVersions: [
     {
       dnpName: "prysm-prater.dnp.dappnode.eth",
       version: "0.1.7"
     }
   ],
 
-  // Web3signer dnpname
-  web3SignerDnpName: "web3signer-prater.dnp.dappnode.eth",
+  // Web3signer and clients dnpnames
+  web3SignerClientsDnpNames: [
```

Comment:
This is really a package exclusion list. A package excludes some other packages from being installable

Reply:
thats correct, a package within a version range exclude the installation of the web3signer for the network of the package